builtin/audit/socket/backend.go

```diff
@@ -428,10 +428,29 @@ func (b *Backend) configureSinkNode(name string, address string, format string,
 		return fmt.Errorf("%s: error creating socket sink node: %w", op, err)
 	}
 
-	sinkNode := &audit.SinkWrapper{Name: name, Sink: n}
+	// Wrap the sink node with metrics middleware
+	sinkMetricTimer, err := audit.NewSinkMetricTimer(name, n)
+	if err != nil {
+		return fmt.Errorf("%s: unable to add timing metrics to sink for path %q: %w", op, name, err)
+	}
+
+	// Decide what kind of labels we want and wrap the sink node inside a metrics counter.
+	var metricLabeler event.Labeler
+	switch {
```

Comment:
nit: Unless we're planning on expanding on the cases in the future, this might be slightly more readable as an if/else statement.

Reply:
Yeah, I know what you mean, I think I ended up liking the switch more as the 'standard' way this thing works is to get a `MetricLabelerAuditSink` so having a `default` case (to me) felt a bit more like: OK. this is the norm and the fallback is some special case. I can change it if it's really bugging you.

Reply:
It wasn't!